.github/CODEOWNERS

```diff
@@ -1,2 +1,40 @@
 # Default code owners for entire repository
-*   @agadzhalov @thenswan @SimiHunjan
+*   @agadzhalov @thenswan @SimiHunjan @hashgraph/release-engineering @hashgraph/release-engineering-managers
+
+#########################
+#####  Core Files  ######
+#########################
+
+# NOTE: Must be placed last to ensure enforcement over all other rules
+
+# Protection Rules for Github Configuration Files and Actions Workflows
+/.github/                                       @hashgraph/release-engineering @hashgraph/release-engineering-managers @0xivanov @MrValioBg
```

Comment:
why would we have explicitly **0xivanov** and **MrValioBg** here? I don't see any reason for that. Also **MrValioBg** is not part from our SDK team so he shouldn't be included